lotus-soup/init.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 func init() {
-	build.BlockDelaySecs = 2
+	build.BlockDelaySecs = 1
```

Comment:
Is this intentional?

Reply:
Not really, just wanted the test to be faster. Will revert. (I didn't see any difference between 2 and 1.)